src/validation.py

```diff
@@ -57,16 +68,23 @@ def check_allocated_gf_matches_input_gf(pudl_out, gen_fuel_allocated):
             "fuel_consumed_for_electricity_mmbtu",
         ]
     ].sum()
-    # calculate the difference between the values
-    plant_total_diff = plant_total_gf - plant_total_alloc
-    # flag values where the absolute difference is greater than 10 mwh or mmbtu
+    # calculate the percentage difference between the values
+    plant_total_diff = (plant_total_alloc - plant_total_gf) / plant_total_gf
+    # flag rows where the absolute percentage difference is greater than our threshold
     mismatched_allocation = plant_total_diff[
```

Comment:
The built-in python function isclose (https://docs.python.org/3/library/math.html#math.isclose) would probably be a bit clearer (and more efficient)

Reply:
not a priority to rewrite, just letting you know it exists for next time.

Reply:
Ok this was a quick update so I changed it to use isclose